client/elements/sc-donate-now-page.html

```diff
@@ -348,6 +352,26 @@
                   },
                   processingPayment: {
                     type: Boolean,
+                  },
+                  errorMessage: {
+                    type: String,
+                    value: 'Error!'
+                  },
+                  StripeValid: {
```

Comment:
1. It's not camelCase
2. Booleans should be named with the appropriate prefix (`isStripeValid`)